docs/html/topics/workflow.md

```diff
@@ -0,0 +1,21 @@
+# Pip is not a workflow tool
```

Comment:
How about this, as I think it's a more commonly used term?

```suggestion
# Pip is not a workflow automation tool
```
Or:

```suggestion
# Pip is not a workflow management tool
```

Reply:
I agree, I'm not sure what it means to be or not be a "workflow tool". Pip has a workflow for the actions it can do.